packages/designmanual/stories/basic-styles.jsx

```diff
@@ -1602,6 +1602,14 @@ storiesOf('Grunnstiler', module)
           typiske sildestimen alle absolutt skal følge, er kult. <br />
           (Jente 19 år, Si;D, Aftenposten 23. mai 2016)
         </blockquote>
+        <h2>Sitat i liste</h2>
```

Comment:
Veldig pirk, men denne kunne også vore plassert under lister.
Pluss:
```suggestion
        <h2>Sitat med liste</h2>
```
Eller:
```suggestion
        <h2>Liste i sitat</h2>
```